packages/model-viewer/src/features/animation.ts

```diff
@@ -85,13 +90,13 @@ export const AnimationMixin = <T extends Constructor<ModelViewerElementBase>>(
       this.dispatchEvent(new CustomEvent('pause'));
     }
 
-    play() {
+    play(options: PlayAnimationOptions = {repetitions: Infinity, pingpong: false}) {
       if (this[$paused] && this.availableAnimations.length > 0) {
         this[$paused] = false;
         this[$renderer].threeRenderer.shadowMap.autoUpdate = true;
 
         if (!this[$scene].hasActiveAnimation) {
-          this[$changeAnimation]();
+          this[$changeAnimation](options);
```

Comment:
I think you're going to need to change this logic. When I `pause` and then `play({repetitions: 1})`, it loops because this line does not get hit. Likewise if it's already playing. You'll want to adjust your tests to catch this too.

Reply:
I removed the part that validates `hasActiveAnimation` in order to execute `this[$changeAnimation]()`.
To be honest, I don't know why we need to validate `hasActiveAnimation`.
I also removed the execution of `[$changeAnimation]` which seems unnecessary.